### PR TITLE
FWI-2850: Changed how we get pod owner

### PR DESCRIPTION
**Pull request by jdesouza**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### CODE REVIEW

File: plugins/prometheus/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 1.2.0
```
Comment (jdesouza) [inline plugins/prometheus/CHANGELOG.md:3]: Maybe 2.0.0 ?

File: plugins/prometheus/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-1.1.0
+1.2.0
```
Comment (jdesouza) [inline plugins/prometheus/version.txt:1]: maybe 2.0.0?

File: plugins/prometheus/CHANGELOG.md (hunk 1 above)
Comment (rbren) [reply plugins/prometheus/CHANGELOG.md:3]: I don't think it's a breaking change, so long as the file format is the same

Comment (jdesouza) [reply plugins/prometheus/CHANGELOG.md:3]: I see

**Comment by rbren** [comment]
@jdesouza I've asked the FWAAC team to use this branch for our own internal prometheus install, so we can test to make sure it looks good.

**Comment by jdesouza** [comment]
:awesome:

**Review by rbren — APPROVED** [review]

**Pull request merged**